influxdb/1.2/alpine/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM alpine:3.5
 
 RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
 
+RUN apk add --no-cache curl bash
```

Comment:
I've removed the `curl` dependency and replaced it by using the `influx` cli, but I'm still not sure if I can make the script `sh` compatible and remove the `bash` dependency, too.